### DISCUSSION TITLE
No tutorials to put this on my bluehost server

### DISCUSSION THREAD

**Discussion by L00ncan** [mistic100/Photo-Sphere-Viewer#1172] [General]
I'm trying to find tutorials how to put this on my bluehost server but can't find much information, i need your application running on my server but cannot find a way, is it as easy as just putting the folder on my server and it works seamlessly however i'm assuming it's not as simple using your application but i cannot find any information. I can use youtube videos and google bard and ask it how to use yarn on my webserver but there's little information of how to place this on my webserver, it's like having an IKEA studio set with lost instructions, i'm lost, you can't just respond with **"learn how to code"** those are unfair responses.

Pannellum and marzipano _360 web viewers_ come with documentation to assure it can be installed easily on webservers, yours does not. 

Can we or can we not have your application running on our websites as advertised?

**Comment by mistic100** [comment]
Sorry this is a JavaScript library, not a complete ready-to-use product. And I am not willing to push it beyond that point, it is already a lot of work to maintain this alone. So, yes, you will have to "learn how to code".

> Can we or can we not have your application running on our websites as advertised?

I am curious to know how you read that.

**Comment by L00ncan** [comment]
Your application comes up on google as the top five 360 web viewer applications manytimes surpassing Pannellum and marzipano, why does it have such a high ranking when it has high next to impossible learning curves for webserver use, even Google Ai is confused when i propose how to get this working with a webserver using yarn, this application even confuses Googl Ai.

Getting started guide has as few words 30 words in a contemptible manner if it's describing help, like an IKEA help blankbook guide only showing the finished installation
![ikea](https://github.com/mistic100/Photo-Sphere-Viewer/assets/89498490/749b91bd-5ddd-428a-a69f-0927544594de)

**Reply by mistic100** [reply]
I do not work for Google, I can't tell you. I am sorry you didn't found what you expected here but if you stay only to rant you will be blocked.

Feel free to experiment with the various demos on the documentation website and the examples folder, and open pertinent support issues if needed. ("Do that for me" issues will be closed).

Have a nice day.

**Reply by L00ncan** [reply]
You're claiming that i'm decalring there's no documentation is a "rant" it's an accurate statement that there is no documentation with this application as it dominates the google rankings for web 360 players
